cli/cli/cmds_volume.go

```diff
@@ -179,7 +178,7 @@ func (c *CLI) initVolumeCmds() {
 		Run: func(cmd *cobra.Command, args []string) {
 			checkVolumeArgs(cmd, args)
 
-			result, err := c.lsVolumes(args)
+			result, err := c.lsVolumes(args, requestAttachments())
```

Comment:
Since we are requesting attachments here, should we be using that info to see if the volume is attached before doing the remove?